tests/support/TempDirectoryTrait.php

```
@@ -0,0 +1,61 @@
+        foreach ($entries as $entry) {
+            if ($entry === '.' || $entry === '..') {
+                continue;
+            }
+
+            $full = $path . '/' . $entry;
+
+            if (is_dir($full)) {
+                $this->removeDirectory($full);
+            } else {
+                unlink($full);
+            }
```

PHPMD (2.15.0)

[error] The method removeDirectory uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)